### DISCUSSION TITLE
Enabling Geotiff File Support in TerriaJS: Is It Possible?

### DISCUSSION THREAD

**Discussion by pabrojast** [TerriaJS/terriajs#6938] [General] [answered]
Hello! I'm currently setting up Terria and I have a question regarding its support for geotiff files. Is there a way to enable Geotiff file support in TerriaJS at present?

**Comment by techmavengeospatial** [comment, marked as answer]
COG Tile Server, GeoSever, QGIS Server, MapServer, or GeoTIFF.js

**Comment by techmavengeospatial** [comment]
https://github.com/TerriaJS/terriajs/pull/6811
https://github.com/TerriaJS/terriajs/pull/6748
https://github.com/TerriaJS/terriajs/pull/6833

**Comment by pabrojast** [comment]
Thank You!
